src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -414,7 +414,7 @@ class VAMap extends Component {
         <div>
           {r.type === LocationType.CC_PROVIDER ? (
             <div>
-              <a onClick={linkAction.bind(this, r.id, true)}>
+              <a href={`/provider/${r.id}`}>
```

Comment:
I removed the `onClick` event handler because it appears it's using router context to change the URL. This works, but won't accept keyboard focus without an href.

Reply:
If the original intent was to load the route without a full page load, maybe this should be a ``<Link to={`/provider/${r.id}`}``, which can be imported from `react-router`. It will still render as a `<a href={...}>`, but do the desired router action.